openhands/server/github.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+import httpx
+
+from openhands.core.logger import openhands_logger as logger
```

Comment:
Without
```py
from dotenv import load_dotenv

load_dotenv()
```

`os.getenv('GITHUB_USER_LIST_FILE')` would return `None` which would mean all users are authorized according to `authenticate_github_user`

Reply:
Why do you say that? That's only if the env vars are in a `.env` file

Reply:
Oh yes